README.md

```diff
@@ -128,6 +129,10 @@ See [http://strong-pm.io](http://strong-pm.io) for more information.
 For more resources, including links to blogs, see
 [http://strong-pm.io/resources/](http://strong-pm.io/resources/).
 
+## Troubleshooting
+
+**Q: On deploy getting an `git: fatal: unable to access 'http://127.0.0.1:8701/default/': Empty reply from server**
+A: Check to make sure `git` is installed
```

Comment:
Needs an extra blank line here. The rendered version squishes these on to the same line otherwise.